src/ringct/rctTypes.h

```diff
@@ -368,6 +368,12 @@ namespace rct {
         template<bool W, template <bool> class Archive>
         bool serialize_rctsig_prunable(Archive<W> &ar, uint8_t type, size_t inputs, size_t outputs, size_t mixin)
         {
+          if (inputs >= 0xffffffff)
```

Comment:
Could you elaborate? Is it an integer overflow bug? Didn't find the exact place where the bug gets triggered at a quick glance.

Reply:
It is. I can elaborate later.